src/Components/Server/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,5 @@
 #nullable enable
+Microsoft.AspNetCore.Components.Server.Circuits.CircuitInboundEventContext
+Microsoft.AspNetCore.Components.Server.Circuits.CircuitInboundEventContext.Circuit.get -> Microsoft.AspNetCore.Components.Server.Circuits.Circuit!
+Microsoft.AspNetCore.Components.Server.Circuits.IHandleCircuitEvent
+Microsoft.AspNetCore.Components.Server.Circuits.IHandleCircuitEvent.HandleInboundEventAsync(Microsoft.AspNetCore.Components.Server.Circuits.CircuitInboundEventContext! context, System.Func<Microsoft.AspNetCore.Components.Server.Circuits.CircuitInboundEventContext!, System.Threading.Tasks.Task!>! next) -> System.Threading.Tasks.Task!
```

Comment:
The naming here surprises me. The last I heard we were calling this "inbound activity" rather than "inbound events". Did that change at some point?

I prefer "activity" over "event" to avoid implying this is only about events in either the Blazor sense (e.g., from a DOM event) or in a C# sense (`event`). Or is there some other reason it should be called "event"?

Reply:
I missed that. @SteveSandersonMS is correct, activity sounds better.

Reply:
I'm happy to change the name back to activity. I noticed that we used the term "event" quite a lot in this discussion, and I thought the term might be clearer to the customer. "Activity" to me seems a bit more vague, whereas I think "event" implies that a distinct occurrence caused the code in question to run (and it also kind of implies there's some handler down the chain). But I think it's splitting hairs at that point, and I definitely see how using an overloaded term like "event" could cause confusion.

Reply:
Oops, I posted that comment before seeing Javier's. I'll change it back to "activity"!